### ISSUE TITLE
Can not reproduce the result

### ISSUE THREAD

**Issue by chaoyan1073** [tdeboissiere/DeepLearningImplementations#60]
For dense net.
Could you please give a detailed parameters description which generated the reported result? I used the default parameters except for those specified in your README file.  Also, I changed the number of the epoch of 300. But only got at most 75% train accuracy. I did not change anything else. What is the matter?

**Comment by chaoyan1073** [comment]
When using SGD as the optimizer, the learning rate should be 0.1, while Adam 0.001.
